odoo/addons/base/tests/test_cli.py

```diff
@@ -134,3 +134,19 @@ def test_shell(self):
             ">>> Hello from Python!",
             '>>> '
         ])
+
+
+class TestCommandUsingDb(TransactionCase):
+
+    def cut_lines(self, buffer, n=5):
+        """ Cut before splitting """
+        max_line_len = 512
+        return buffer[:max_line_len * n].split('\n')[:n]
+
+    def test_i18n_export(self):
+        proc = run_command('i18n', 'export', '-d', self.env.cr.dbname, '-o', '-', 'base')
+        self.assertEqual(self.cut_lines(proc.stdout, n=3), [
+            '# Translation of Odoo Server.',
+            '# This file contains the translation of the following modules:',
+            '# \t* base'
+        ])
```

Comment:
no need to capture the entire output if all we want are the first 3 lines
```suggestion
        proc = popen_command('i18n', 'export', '-d', self.env.cr.dbname, '-o', '-', 'base')
        self.assertEqual(proc.wait(), 0, proc.stderr.read())
        self.assertEqual(proc.readline(), '# Translation of Odoo Server.')
        self.assertEqual(proc.readline(), '# This file contains the translation of the following modules:')
        self.assertEqual(proc.readline(), '# \t* base')
```